lib/universal-search.js

```diff
@@ -70,6 +74,24 @@ Search.prototype = {
     }
   },
 
+  _enableSearchSuggestions: function(win) {
+    // Enable the browser.search.suggest.enabled pref. Save the previous state
+    // so we can restore it on unload.
+    const searchSuggestIsEnabled = Services.prefs.getBoolPref('browser.search.suggest.enabled');
```

Comment:
I think the pref we want is `browser.urlbar.suggest.searches`. Have a look at [this line](https://github.com/mozilla/gecko-dev/blame/8832806e/testing/profiles/prefs_general.js#L332), seems like that's the pref that was introduced by the UnifiedComplete work.

Because this pref was recently added, it's possible that it won't exist in a given profile. Unfortunately, `getBoolPref` will throw if the pref doesn't exist, so you'll need to use `Services.prefs.getPrefType` to safely check for the pref's existence first.